apm-agent-plugins/apm-jdbc-plugin/src/test/java/co/elastic/apm/agent/jdbc/AbstractJdbcInstrumentationTest.java

```diff
@@ -195,6 +198,25 @@ private void testUpdate(boolean isLargeUpdate) throws SQLException {
         assertSpanRecorded(insert, false, 1);
     }
 
+    private void testPreparedStatementUpdate(boolean isLargeUpdate) throws SQLException {
+        final String insert = "INSERT INTO ELASTIC_APM (FOO, BAR) VALUES (42, 'TEST')";
```

Comment:
this is not an update 🤔 
but it's just about testing `executeUpdate()` right?

Reply:
This API is for DML queries in general, not specific to updates